### DISCUSSION TITLE
Initial Release V2.1 VICEWIZE HD Italic Color Font Set

### DISCUSSION THREAD

**Discussion by vicewize** [vicewize/vicewizeosdfontset#2] [General]
A full color HD optimized DJI overlay for MSP-OSD by WTF-FPV. All artwork made by hand in Illustrator and Photoshop, and issues feel free to let me know.

_VICEWIZE
www.facebook.com/efceskoltd

<a href="https://ibb.co/0hrbY9Y"><img src="https://i.ibb.co/D5Q2wMw/template.png" alt="template" border="0"></a>

<hr /><em>This discussion was created from the release <a href='https://github.com/vicewize/vicewizeosdfontset/releases/tag/2.1'>Initial Release V2.1 VICEWIZE HD Italic Color Font Set</a>.</em>

**Comment by aleksei85m** [comment]
Hi,
You did great work! Your font/OSD is awesome!
Is there a chance that a "Not Italic" version will appear in the future?
